graphql.gemspec

```diff
@@ -41,6 +41,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency "sequel"
   s.add_development_dependency "sqlite3"
 
+  # TODO(xuorig) probably can in house it:
+  s.add_runtime_dependency "activesupport"
```

Comment:
:+1: since we're only using one function, we can probably avoid forcing activesupport on people.

One option would be to use a built-in implementation by default, but support user-provided transforms, something like 

```
rename :camelize # => uses the built-in transformation 
rename MyCustomRename # => uses `.call(name)` to get a new name 
```

That way, people who _want_ some more extensive inflector features could still use them. (For example, it looks like ActiveSupport tries to detect acronyms.)

Another option would be to do both: If ActiveSupport is present, use it, otherwise, use a local implementation. I wonder if being "smart" would just be annoying though, if people noticed it behaving differently at different times.